git_repo/repo.py

```diff
@@ -426,17 +426,20 @@ def request_edition(repository, from_branch, onto_target):
 
         service = self.get_service(resolve_targets=('upstream', '{service}', 'origin'))
 
-        new_request = service.request_create(self.namespace,
+        output_generator = service.request_create(self.namespace,
                 self.repo_name,
                 self.local_branch,
                 self.remote_branch,
                 self.title,
                 self.message,
                 self._auto_slug,
                 request_edition)
-        log.info('Successfully created request of `{local}` onto `{project}:{remote}`, with id `{ref}`!'.format(**new_request))
-        if 'url' in new_request:
-            log.info('available at: {url}'.format(**new_request))
+
+        log.info(next(output_generator))
+        fmt = next(output_generator)
+        for item in output_generator:
+            log.info(fmt.format(**item))
+
```

Comment:
you can simplify that using the [`print_iter`](https://github.com/guyzmo/git-repo/blob/a724e1ae0ceaa7651a33320097b9499e88f13789/git_repo/tools.py#L10) function I wrote in the `.tools` module! Then all you got to do is:

```python
print_iter(service.request_create(…))
```

Reply:
Okay. Changed that